docs/node/run-your-node/prerequisites/set-up-trusted-execution-environment-tee.md

```diff
@@ -307,9 +306,9 @@ support DCAP attestation, you'll need to migrate your node to newer hardware.
       ```bash
       sudo systemctl restart aesmd.service
       ```
-4. [Configure the Quote Provider].
-5. Use the [attestation tool] to test if your settings are correct.
-6. Start your compute node.
+3. [Configure the Quote Provider].
+4. Use the [attestation tool] to test if your settings are correct.
+5. Restart your compute node and verify that node has registered.
```

Comment:
It would be better to check if it is "ready" (it could happen that it is "registered", but not "ready").